src/common/cont_props.c

```diff
@@ -224,7 +224,7 @@ daos_cont_prop2redunlvl(daos_prop_t *props)
 	struct daos_prop_entry *prop =
 		daos_prop_entry_get(props, DAOS_PROP_CO_REDUN_LVL);
 
-	return prop == NULL ? DAOS_PROP_CO_REDUN_RANK : (uint32_t)prop->dpe_val;
```

Comment:
[suggestion] "#define DAOS_PROP_CO_REDUN_DEFAULT DAOS_PROP_CO_REDUN_NODE", replace the other DAOS_PROP_CO_REDUN_NODE usages with DAOS_PROP_CO_REDUN_DEFAULT. Then in the future, if we change the default domain as others (may be new), then we can just redefine DAOS_PROP_CO_REDUN_DEFAULT.